.github/actions/deploy-portal/action.yml

```diff
@@ -1,75 +1,74 @@
-name: Deploy portal
+name: Deploy Portal
+
 description: Deploys the portal to Hostinger
 
 inputs:
   HOSTINGER_HOST:
     required: true
-    description: 'Hostinger host'
+    description: Hostinger host
   HOSTINGER_PORT:
     required: true
-    description: 'Hostinger port'
+    description: Hostinger port
   HOSTINGER_SSH_KEY:
     required: true
-    description: 'Hostinger SSH key'
+    description: Hostinger SSH key
   HOSTINGER_USER:
     required: true
-    description: 'Hostinger user'
+    description: Hostinger user
   HOSTINGER_TARGET:
     required: true
-    description: 'Hostinger target'
+    description: Hostinger target
   NEXT_PUBLIC_CUSTOM_RPC_URL_HEMI_MAINNET:
     required: false
-    description: 'Custom RPC URL for Hemi Mainnet'
+    description: Custom RPC URL for Hemi Mainnet
   NEXT_PUBLIC_CUSTOM_RPC_URL_HEMI_SEPOLIA:
     required: false
-    description: 'Custom RPC URL for Hemi Sepolia'
+    description: Custom RPC URL for Hemi Sepolia
   NEXT_PUBLIC_CUSTOM_RPC_URL_MAINNET:
     required: false
-    description: 'Custom RPC URL for Mainnet'
+    description: Custom RPC URL for Mainnet
   NEXT_PUBLIC_CUSTOM_RPC_URL_SEPOLIA:
     required: false
-    description: 'Custom RPC URL for Sepolia'
+    description: Custom RPC URL for Sepolia
   NEXT_PUBLIC_FEATURE_FLAG_ENABLE_BTC_TUNNEL:
     required: true
-    description: 'Feature flag to enable BTC tunnel'
+    description: Feature flag to enable BTC tunnel
   NEXT_PUBLIC_RECAPTCHA_SITE_KEY:
     required: true
-    description: 'Recaptcha site key'
+    description: Recaptcha site key
   NEXT_PUBLIC_CLAIM_TOKENS_URL:
     required: true
-    description: 'Claim tokens microservice URL'
+    description: Claim tokens microservice URL
 
 runs:
   using: composite
   steps:
-    - name: Build site
-      run: npx lerna run deploy --scope=app
+    - run: npx lerna run deploy --scope=app
       shell: bash
+      name: Build site
```

Comment:
I think `name` should go before `shell`?

Both by natural order and our guideline order which recommends following [this order](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsname)

![image](https://github.com/user-attachments/assets/42cf93cf-8082-4ccd-87c6-42eb1ba757d5)

Reply:
No. The action metadata syntax docs have a different order than the workflow docs. 🤷🏻‍♂️

https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#runs-for-composite-actions

I will update the docs.

Reply:
Done!

https://github.com/bloqpriv/software-development-process/blob/master/guides/GitHub%20Actions.md#composite-action-guidelines